src/Aspire.Hosting.OpenAI/OpenAIResource.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.OpenAI;
+
+/// <summary>
+/// Represents a logical OpenAI account/configuration that can host one or more <see cref="OpenAIModelResource"/> children.
+/// </summary>
+public sealed class OpenAIResource : Resource, IResourceWithoutLifetime, IResourceWithConnectionString
```

Comment:
Do we want IResourceWithoutLifetime?

Reply:
Updated

<img width="1279" height="275" alt="image" src="https://github.com/user-attachments/assets/1977088e-bac9-4fc7-bf46-3f62172fa5cb" />

```
  "status": {
    "description": "Partial System Degradation",
    "indicator": "minor"
  }
```

Should the health check be opt-in? Or is there a way to remove it that is not manual in case it prevents the model from starting because an unrelated component?

Reply:
Not sure, we made it opt in for GitHub models because of rate limiting yea?

Reply:
This one is fine. The one on the model is still opt-in, but honestly this could be by default too since it's a single request whose result is cached. In GH models every single request count because some models literally reach the limit at 1 or two requests.